nav2_bringup/bringup/params/nav2_params.yaml

```diff
@@ -268,6 +268,9 @@ planner_server:
       change_penalty: 0.20              # For SE2 node: penalty to apply if motion is changing directions, must be >= 0
       non_straight_penalty: 1.10        # For SE2 node: penalty to apply if motion is non-straight, must be => 1
       cost_penalty: 1.3                 # For SE2 node: penalty to apply to higher cost zones
+      analytic_expansion_ratio: 2.0     # For SE2 node: planner will attempt an analytic path expansion every N iterations
```

Comment:
Please add to the parameters guide; https://github.com/SteveMacenski/navigation2/blob/nav2_smac_planner/doc/parameters/param_list.md#smac_planner

Reply:
I took care of this param for the website docs

Reply:
Done